src/Aacotroneo/Saml2/Saml2Auth.php

```diff
@@ -99,8 +100,13 @@ function sls()
     {
         $auth = $this->auth;
 
-        $keep_local_session = true; //we don't touch session here
-        $auth->processSLO($keep_local_session);
+        // destroy the local session by firing the Logout event
+        $keep_local_session = false;
+        $session_callback = function () {
+            \Event::fire(new Saml2LogoutEvent());
+        };
+
+        $auth->processSLO($keep_local_session, null, false, $session_callback);
```

Comment:
Either way it will _attempt_ to destroy the local session, but if we don't provide the callback it will attempt to destroy the local session with `session_unset()` (effectively). If `$keep_local_session` is true, then the callback isn't called.

This is consistent with how their python library works.